### PR TITLE
Render navigation and addons panels even when they are hidden

### PR DESCRIPTION
**Pull request by Hypnosphi**
Issue: #2280 

Some addons (backgrounds, notes, probably others) send data from preview to addon panel when the former is rendered. The problem is that if the panel is hidden at that point, it's not actually rendered, so it will never receive that data.

## What I did
Render addon panel with `display: none` when it should be hidden. Do the same with navigation panel just for consistency.

## How to test
Open [backgrounds](https://deploy-preview-2336--storybook-docs.netlify.com/cra-kitchen-sink/?selectedKind=Addon%20Backgrounds&selectedStory=story%201&full=0&down=0&left=1&panelRight=1&downPanel=storybook-addon-background%2Fbackground-panel) and [notes](https://deploy-preview-2336--storybook-docs.netlify.com/cra-kitchen-sink/?selectedKind=Addon%20Notes&selectedStory=withNotes&full=0&down=1&left=1&panelRight=1&downPanel=storybook%2Fnotes%2Fpanel) examples in cra-kitchen-sink, toggle addon panel with `⌘ ⇧ D`

### CODE REVIEW

**Comment by Spy-Seth** [comment]
I love this! We have currently the issue with it with some in house addons! If you need some help/feedback do not hesitate 😄

File: lib/ui/src/modules/ui/components/layout/index.test.js [hunk 1]
```diff
@@ -17,15 +19,12 @@ describe('manager.ui.components.layout.index', () => {
         />
       );
 
-      const markup = wrap.html();
-      expect(markup).toMatch(/LeftPanel/);
-      expect(markup).toMatch(/DownPanel/);
-      expect(markup).toMatch(/Preview/);
+      expect(wrap).toMatchSnapshot();
```
Comment (ndelangen) [inline lib/ui/src/modules/ui/components/layout/index.test.js]: This snapshot is huge, is there some sane way to deal with that, or is it inevitable?

File: lib/ui/src/modules/ui/components/layout/index.test.js [hunk 2]
```diff
@@ -38,10 +37,7 @@ describe('manager.ui.components.layout.index', () => {
         />
       );
 
-      const markup = wrap.html();
-      expect(markup).not.toMatch(/LeftPanel/);
-      expect(markup).not.toMatch(/DownPanel/);
-      expect(markup).toMatch(/Preview/);
+      expect(wrap).toMatchSnapshot();
```
Comment (ndelangen) [inline lib/ui/src/modules/ui/components/layout/index.test.js]: same

File: lib/ui/src/modules/ui/components/layout/index.test.js [hunk 3]
```diff
@@ -59,10 +55,7 @@ describe('manager.ui.components.layout.index', () => {
         />
       );
 
-      const markup = wrap.html();
-      expect(markup).not.toMatch(/LeftPanel/);
-      expect(markup).toMatch(/DownPanel/);
-      expect(markup).toMatch(/Preview/);
+      expect(wrap).toMatchSnapshot();
```
Comment (ndelangen) [inline lib/ui/src/modules/ui/components/layout/index.test.js]: same

File: lib/ui/src/modules/ui/components/layout/index.test.js [hunk 4]
```diff
@@ -80,10 +73,7 @@ describe('manager.ui.components.layout.index', () => {
         />
       );
 
-      const markup = wrap.html();
-      expect(markup).toMatch(/LeftPanel/);
-      expect(markup).not.toMatch(/DownPanel/);
-      expect(markup).toMatch(/Preview/);
+      expect(wrap).toMatchSnapshot();
```
Comment (ndelangen) [inline lib/ui/src/modules/ui/components/layout/index.test.js]: same

File: lib/ui/src/modules/ui/components/layout/index.test.js (hunk 1 above)
Comment (Hypnosphi) [reply lib/ui/src/modules/ui/components/layout/index.test.js]: We can explicitly test the presence of particular styles instead

Comment (Hypnosphi) [reply lib/ui/src/modules/ui/components/layout/index.test.js]: Did that

**Review by ndelangen — APPROVED** [review]

**Pull request merged**